examples/lighting-app/efr32/BUILD.gn

```diff
@@ -123,7 +123,11 @@ efr32_executable("lighting_app") {
     ldscript = "${efr32_project_dir}/ldscripts/efr32-light-example-MG21.ld"
   }
 
-  ldflags = [ "-T" + rebase_path(ldscript, root_build_dir) ]
+  ldflags = [
+    "-Wl,--print-memory-usage",
```

Comment:
Can you move this behind a build argument

```
if (chip_print_memory_usage) {
  ldflags += [ ... ]
}
```

Console output should be an opt-in (or written to a file) to keep signal/noise high.

Reply:
Done, Thank you